portfolio/src/main/webapp/index.html

```diff
@@ -2,18 +2,259 @@
+                        <li class="list-group-item bg-success text-center text-w text-light h5">Python</li>
+                        <li class="list-group-item bg-success text-center text-w text-light h5">C#</li>
+                        <li class="list-group-item bg-success text-center text-w text-light h5">HTML/CSS/JavaScript</li>
+                        <li class="list-group-item bg-success text-center text-w text-light h5">Elixir/Racket</li>
+                        <li class="list-group-item bg-success text-center text-w text-light h5">Unity</li>
+                        <li class="list-group-item bg-success text-center text-w text-light h5">Solidworks</li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- Random quote section -->
+    <section class="p-5 bg-primary text-center text-md-start text-light" id="randomQuote">
+        <div class="container bg-primary text-light">
+            <div class="row align-items-center justify-content-between">
+                <!-- Text section -->
+                <div class="col-md">
+                    <h1 class="pb-3">Random Quote :P</h1>
+                    <p>If you're bored, hear a random quote from my favourite horror movies.</p>
+                </div>
+                <!-- Quote section -->
+                <div class="col-md lead text-center">
+                    <button type="button" class="btn btn-warning" onclick="addQuote()"><i class="bi bi-chat-square-quote text-light"></i></button>
+                    <p class="pt-2" id="quote"></p>
+                </div>
+            </div>
+        </div>
+    </section>
+
+    <!-- Contact me section -->
+    <section class="p-5 text-center" id="contactMe">
+        <div class="container">
+            <h1 class="text-center pb-3">Contact Me :D</h1>
+            <p class="lead">Here's some sites where you can follow or contact me. <br> Please I'm desperate.</p>
+            <div class="row text-center g-4">
+                <!-- GitHub card -->
+                <div class="col-md d-flex align-items-stretch">
+                    <div class="card bg-light text-dark">
+                        <div class="card-body text-center">
+                            <div class="h1 mb-3">
+                                <i class="bi bi-linkedin"></i>
+                            </div>
+                            <h3 class="card-title">
+                                Linkedin
+                            </h3>
+                            <p class="card-text">
+                                I just recently made a Linkedin account, so its pretty empty right now. Still I would love to make connections.
+                            </p>
+                            <button href="https://www.linkedin.com/in/mateo-herrera-lavalle-6059a9239" role="button" class="btn btn-outline-primary">Go Here</button>
```

Comment:
[3.1.2](https://google.github.io/styleguide/htmlcssguide.html#HTML_Validity): Use valid HTML where possible.

Same for the GitHub button.

You can use tools such as the [W3C HTML validator](https://validator.w3.org/nu/) to copy/paste your HTML and check for errors.

You might want to wrap this around an `<a>` element instead.

BTW, I don't think the `role="button"` is necessary as this is already a `<button>` element.